src/core/solver.py

```
@@ -108,67 +123,96 @@ class ScoringStrategy(ABC):
-        """Returns (name, scores, weight_multiplier) for each dimension."""
-        pass
+        """Returns score vectors for each dimension to be optimized.
+
+        Args:
+            participants (list[Participant]): List of participants.
+            cfg (SolverConfig): Solver configuration.
+
+        Returns:
+            list[tuple[str, list[int], int]]: List of (name, scores, weight) tuples.
+        """
+        pass  # pragma: no cover
```

Ruff (0.15.12)

[warning] Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)